plugins/applications/src/lib.rs

```diff
@@ -65,7 +65,12 @@ pub fn handler(selection: Match, state: &State) -> HandleResult {
                 }
             }
         }
-    } else if let Err(why) = Command::new("sh").arg("-c").arg(&entry.exec).spawn() {
+    } else if let Err(why) = Command::new("sh")
+        .arg("-c")
+        .arg(&entry.exec)
+        .current_dir(entry.path.as_ref().unwrap_or(&env::current_dir().unwrap()))
```

Comment:
Indentation seems a bit wonky here, format the code with `cargo fmt`.

Reply:
I ran `cargo fmt` and it's still the same